### DISCUSSION TITLE
Does pagination support skipping/offsetting?

### DISCUSSION THREAD

**Discussion by AleksandrHovhannisyan** [11ty/eleventy#1843] [Q&A] [answered]
I'm working on migrating my [Jekyll site](https://www.aleksandrhovhannisyan.com/) to 11ty and am so far pleased with how pagination is built into the framework! Very cool that you can reuse pagination in front matter for permalinks.

I have a unique need: I want `/blog/` to show the 20 most recent posts and to be treated as Page 1 implicitly—i.e., I don't want its permalink to be `/blog/page/1/` because that's a bit ugly for sharing. When you navigate to Page 2, I want the URL to update to `/blog/page/2/`.

So far, I've gotten very close to achieving this with the following directory structure:

```
src/_pages
├── _pages.json
├── blog
│   ├── index.html
│   └── page
│       └── index.html
```

The root blog index does this:

```liquid
---
title: Blog
layout: blog
permalink: /blog/
---

{%- comment -%}First 20 posts on /blog/{%- endcomment -%}
{%- assign posts = collections.post | limit: 20 -%}
{% for post in posts  %}
  <li><a href="{{ post.url }}">{{ post.date | date: '%b %d, %Y' }} - {{ post.data.title }}</a></li>
{% endfor %}
```

And the nested page index is paginated like so:

```liquid
---
title: Blog (Page {{ pagination.pageNumber }})
layout: blog
permalink: "/blog/page/{{ pagination.pageNumber | plus: 1 }}/"
pagination:
  data: collections.post
  size: 20
  alias: posts
---

{% for post in posts %}
  <li><a href="{{ post.url }}">{{ post.date | date: '%b %d, %Y' }} - {{ post.data.title }}</a></li>
{% endfor %}
```

Now, I'm also hoping to get rid of the duplicate `/blog/page/1/` since that's already taken care of by the root `/blog/` page.

Is there any way I can tell the paginator to skip the first page?

**Comment by pdehaan** [comment, marked as answer]
I think https://www.11ty.dev/docs/pagination/#remapping-with-permalinks might have some pointers.

**Reply by AleksandrHovhannisyan** [reply]
Oh, that's awesome! I was actually reading that, but I missed the bit about being able to use conditionals in front matter. 

Jekyll never let me do that, so this is really neat!
